webpackfile.js

```diff
@@ -26,73 +26,95 @@ const nestBlacklist = [
   '^redis$',
 ];
 
+const optimization = {
+  minimize: true,
+  minimizer: [new TerserPlugin()],
+  noEmitOnErrors: true,
+  nodeEnv: 'production',
+  occurrenceOrder: true,
+  providedExports: true,
+  usedExports: true,
+  sideEffects: true,
+  splitChunks: {
+    name: true,
+    cacheGroups: {
+      assets: {
+        test: /node_modules/,
+        chunks: 'all',
+        filename: '[name].server.js',
+      },
+    },
+  },
+};
+
+const mode = 'none';
+
+const wpResolve = {
+  extensions: ['.ts', '.tsx', '.js', '.jsx'],
+  plugins: [
+    new TSConfigPathsPlugin(),
+  ],
+};
+
+const tsLoader = {
+  test: /\.tsx?$/,
+  include: resolve(__dirname, 'src'),
+  exclude: /node_modules/,
+  use: [
+    {
+      loader: 'ts-loader',
+      options: {
+        happyPackMode: true,
+      },
+    },
+  ],
+};
+
 const client = {
   name: 'client',
-  mode: 'production',
+  mode,
   entry: ['./src/client/index.ts'],
   output: {
-    path: resolve(__dirname, 'dist/static'),
+    path: resolve(__dirname, 'build/static'),
```

Comment:
What's the difference between `build/` and `dist/`?

Reply:
None, really. I think the typescript default is `dist/`, but most of our other configurations use `build/`, particularly in our `Dockerfile` where [it copies the `build` dir from the `builder` stage][dockerfile] into the production image. As you would expect, trying to copy from a non-existent directory doesn't work. 

[dockerfile]: https://github.com/seas-computing/react-boilerplate/blob/64c36ae69a61333a4a2ca6ec5376c33165935f9f/Dockerfile#L24

Reply:
Fair enough. I just wondered if there was anything different going on here :)